app/app.py

```diff
@@ -251,27 +251,34 @@ def main() -> None:
                     full_response = ""
                     message_placeholder = st.empty()
 
-                    # Query vector store
-                    context = query_vector_store(
-                        chroma_client=chroma_client,
-                        query_text=prompt,
-                        collection_name=COLLECTION_NAMES[0],
-                        n_results=N_CLOSEST_MATCHES,
-                        embedding_function=embed_function,
-                    )
+                    response = {}
+
+                    for collection in COLLECTION_NAMES:
```

Comment:
```diff
- for collection in COLLECTION_NAMES
+ for collection_name, data in COLLECTION_NAMES.items()
```

Reply:
Looks like `COLLECTION_NAMES` is a list.

Reply:
updated my suggestion.

Reply:
Will `source` be used for anything?